spec/components/markdown_editor_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe "Markdown editor", type: :view do
```

Comment:
Thanks for adding the tests! Can you fix them up to the first commit, where the component is added?

Reply:
Sure, figured it would be easier to review what's been changed since the last review. Fixed-up now to the first commit and rebased to master.